smart-send-logistics/build/index.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array('wp-plugins'), 'version' => '3f3d50b7ebd74f8ef2af');
```

Comment:
```suggestion
<?php

return array(
    'dependencies' => array('wp-plugins'),
    'version' => '3f3d50b7ebd74f8ef2af',
);

```

Reply:
Where does this version come from?

Reply:
The version Number is coming from our plugin main file